src/icons/icon-info.tsx

```diff
@@ -53,9 +43,13 @@ const IconInfoDefault = (): JSX.Element => {
     );
 };
 
-const IconInfo = (): JSX.Element => {
+const IconInfo: React.FC<Props> = (props) => {
```

Comment:
Please, use the `JSX.Element` return type

```ts
const IconInfo = (props: Props): JSX.Element => { ...
```

some context: https://github.com/facebook/create-react-app/pull/8177

Reply:
fixed